WordPress/Classes/Utility/WPError.m

```diff
@@ -178,7 +178,7 @@ + (void)showAlertWithTitle:(NSString *)title message:(NSString *)message withSup
                                                                
                                                                if ([Feature enabled:FeatureFlagZendeskMobile]) {
                                                                    SupportTableViewController *supportVC = [SupportTableViewController new];
-                                                                   supportVC.sourceTag = sourceTag;
+                                                                   supportVC.sourceTagDescription = sourceTag;
```

Comment:
This was actually mapping the `sourceTag` (String) into a Swift Enum Case. Did you have any issues with this mapping?

If we could revert this change, `SupportVC` could just have `sourceTag` (we could remove `sourceTagDescription`, and simplify a little bit everything!).

Reply:
Just to recap what we discussed, `WordPressSupportSourceTag` is a Swift enum that can't be used in ObjC. So I still need `sourceTagDescription`,  but I will "hide the dirt". 😄